incident/templates/incident/_filters.html

```diff
@@ -2,7 +2,7 @@
 	<header class="filters__header">
 		Filters
 
-		<a href="{{ page.url }}">Clear All</a>
+		<a class="text-link" href="{{ page.url }}">Clear All</a>
```

Comment:
Maybe let's change this from a link to `<input type="reset" value="Clear All" />`. That will avoid the reloading of the page which created some confusion. Though not necessarily need to be part of this PR.